content-blocker-lib-ios/ContentBlockerGenerator/Sources/ContentBlockerGenerator/Entity.swift

```diff
@@ -0,0 +1,10 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0
+
+import Foundation
+
+struct Entity {
+    var properties: [String]
+    var resources: [String]
+}
```

Comment:
nit: would be nice to add what this entity does, could be in your doc :)

Reply:
Yeah I think this suits well to the future documentation, since Entities, resources and properties are part of the knowledge I need to explain. When that's done this will be clear as to what this means.